nntrainer/layers/layer_internal.h

```diff
@@ -273,6 +274,7 @@ class Layer {
     in_length = 33,
     recurrent_activation = 34,
     distribute = 35,
+    split_dimension = 36,
```

Comment:
Wonder if you'd like to try properties. I am willing to reach out if you are interested :)

Reply:
how about it as the first thing on top of this PR?

Reply:
Sure, let me do it.

Reply:
let me do this one, and get adjusted to the updated format :).

Reply:
Awesome :)